src/components/AnalysisPage/AnalysisPage.jsx

```diff
@@ -104,7 +104,15 @@ const AnalysisPage = () => {
         </Col>
         <Col xs={{ span: 24, order: 3 }} lg={{ span: 24, order: 5 }}>
           <div id="unit-selector-parent-small">
-            <UnitsSelector partId={partId} units={units} setUnits={setUnits} />
+            <UnitsSelector
```

Comment:
Do you think theres a better way to handle modifying a component for different screen size than just specifying it twice so that stuff like this doesn't happen? Or does this seem to be the proper way to handle this scenario?

Reply:
I'm not sure if someone who was a front end expert would consider it proper or not. 

But in our scenario, It is proper if you ask me. The large screen UnitsSelector is inside of a column with the geometry analysis box and thickness anaylsis box, which is how we wanted the analysis page to be laid out. 

Without having a seperate small screen UnitsSelector seperate from the column with the geo-box and thickness-box. I wouldn't have been able to put it on the top of the page when viewing from a cell phone.